internal/command/meta_backend_test.go

```diff
@@ -2725,3 +3009,12 @@ func testStateStoreMock(t *testing.T) *testing_provider.MockProvider {
 		},
 	}
 }
+
+func configBodyForTest(t *testing.T, config string) hcl.Body {
+	t.Helper()
+	f, diags := hclsyntax.ParseConfig([]byte(config), "", hcl.Pos{Line: 1, Column: 1})
```

Comment:
```suggestion
	f, diags := hclsyntax.ParseConfig([]byte(config), "", hcl.InitialPos)
```
https://pkg.go.dev/github.com/hashicorp/hcl/v2@v2.24.0#pkg-variables